Composer/packages/client/src/components/AppUpdater/styles.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { css } from '@emotion/core';
+import { IButtonStyles } from 'office-ui-fabric-react/lib/Button';
+import { IDialogContentStyles, IDialogFooterStyles } from 'office-ui-fabric-react/lib/Dialog';
+import { IModalStyles } from 'office-ui-fabric-react/lib/Modal';
+
+export const optionIcon = checked => css`
+  vertical-align: text-bottom;
+  font-size: 18px;
+  margin-right: 10px;
+  color: ${checked ? '#0078d4' : '#000'};
```

Comment:
Can you use the fluent color variables here? I think `@ui-fabric/fluent-theme` exports them.